src/components/DataTable.vue

```diff
@@ -10,71 +10,65 @@
         </tr>
       </thead>
       <tbody>
-        <tr v-bind:key="postItem + index" v-for="(postItem, index) in postData">
+        <tr
+          v-bind:key="postItem + index"
```

Comment:
Why we are setting `postItem` (which is an `Object`) in key. I think `"postItem-"+index` (type `String`) is enough for uniqueness. Your thoughts ?